yt_dlp/extractor/playsuisse.py

```diff
@@ -134,12 +135,35 @@ class PlaySuisseIE(InfoExtractor):
             id
             url
         }'''
+    _ID_TOKEN = None
+
+    def _perform_login(self, username, password):
+        login_page = self._download_webpage(
+            'https://www.playsuisse.ch/api/sso/login', None, note='Downloading login page',
+            query={'x': 'x', 'locale': 'de', 'redirectUrl': 'https://www.playsuisse.ch/'})
+        settings = self._search_json(r'var\s+SETTINGS\s*=', login_page, 'settings', None)
+
+        webpage, urlh = self._download_webpage_handle(
+            'https://login.srgssr.ch/srgssrlogin.onmicrosoft.com/B2C_1A__SignInV2/SelfAsserted',
+            None, note='Logging in', headers={'X-CSRF-TOKEN': settings['csrf']},
+            data=urlencode_postdata({'request_type': 'RESPONSE', 'signInName': username, 'password': password}),
+            query={'tx': settings['transId'], 'p': 'B2C_1A__SignInV2'})
```

Comment:
we should expect an error if the user passes the wrong credentials
```suggestion
        status = traverse_obj(self._download_json(
            f'{self._LOGIN_BASE_URL}/{self._LOGIN_PATH}/SelfAsserted', None, 'Logging in'
            query=query, headers={'X-CSRF-TOKEN': csrf_token}, data=urlencode_postdata({
                'request_type': 'RESPONSE',
                'signInName': username,
                'password': password
            }), expected_status=400), ('status', {int_or_none}))
        if status == 400:
            raise ExtractorError('Invalid username or password', expected=True)
```